cosmwasm/contracts/outpost/src/lib.rs

```diff
@@ -0,0 +1,7 @@
+pub mod contract;
+mod error;
+pub mod execute;
+pub mod msg;
+pub mod state;
+
+pub use crate::error::ContractError;
```

Comment:
lol the lib top level crate starts feeling a bit silly in current contract code structures lol

Reply:
yeah. I've seen people use it to define the entry points there, which I like. It basically replaces contract.rs